plugins/by-name/kitty-scrollback/default.nix

```diff
@@ -0,0 +1,7 @@
+{ lib, ... }:
+lib.nixvim.plugins.mkNeovimPlugin {
+  name = "kitty-scrollback";
```

Comment:
Do we need to set `packPathName` to `"kitty-scrollback.nvim"`?

(See https://nix-community.github.io/nixvim/CONTRIBUTING.html#mkneovimplugin and [`:h 'packpath'`](https://neovim.io/doc/user/options.html#'packpath'))

Aside: we really should add some way of asserting the correct packpath name is defined :face_with_diagonal_mouth: